README.md

```diff
@@ -37,8 +37,8 @@ If you do not want to setup `node.js` on your machine and instead have docker in
 MOUNT_SRC=$PWD/docs/site/
 
 # where in the url-structure to attach the folder you just selected. without the leading slash.
-# e.g. for konvoy this might be "dkp/konvoy/1.7".
-MOUNT_DST=dkp/konvoy/1.7
+# e.g. for kommander this might be "dkp/konvoy/2.2".
+MOUNT_DST=dkp/kommander/2.2
```

Comment:
I think we can leave this as it was. It will always be a moving target

Reply:
The previous command did not work for me at all though, I received an error, so in case someone tries to do a copypasta I wanted to update. I don't know if there is an issue with the use of 1.7 directly.